utils/python/CIME/XML/env_batch.py

```diff
@@ -105,3 +105,16 @@ def create_job_groups(self, bjobs):
                 newjob.append(deepcopy(child))
             group.append(newjob)
 
+    def cleanupnode(self, node):
```

Comment:
The default cleanupnode in EntryID removes the default_value node, but we want to keep this one.